### PR TITLE
Lesson plans

### PR DESCRIPTION
**Pull request by nbuonin**
My changes are here running here:
http://kang.ccnmtl.columbia.edu/sb/nbuonin/drupal7_mathkids/

You can login at /user and click on content to see new type.  You can review the video player on the video pages.

This PR consists of adding a print style sheet for the new lesson plan content type.   It also replaces the swf video player for an HTML5 player.

The remaining commits are Drupal module updates.

### CODE REVIEW

File: sites/default/themes/mythemes/acq_mathkids/local.js [hunk 1]
```diff
@@ -115,15 +115,32 @@
...
+  Drupal.behaviors.acquia.checkThemeBuilderStatus = function () {
     return $('body').hasClass('themebuilder') ? true : false;
   };
-  
+
+  /**
+   * Sets default poster for video player
+   */
+    Drupal.behaviors.flowplayerOptions = function () {
+        //flowplayer.conf = {
```
Comment (sdreher) [inline sites/default/themes/mythemes/acq_mathkids/local.js]: Can this commented code be removed?

File: sites/default/themes/mythemes/acq_mathkids/template.php [hunk 2]
```diff
@@ -54,27 +54,40 @@ function acq_mathkids_preprocess_page(&$variables) {
...
-  
+
   $variables['sidebars']['left'] = str_split($sidebarsLeft);
   $variables['sidebars']['right'] = str_split($sidebarsRight);
   $variables['sidebars']['hidden'] = str_split($sidebarsHidden);
 
   // add our flowplayer javascript
-  drupal_add_js(drupal_get_path('theme', 'acq_mathkids') . '/js/flowplayer-3.2.8.min.js');
-  drupal_add_js(drupal_get_path('theme', 'acq_mathkids') . '/js/flowplayer.ipad-3.2.8.min.js');
+  // drupal_add_js(drupal_get_path('theme', 'acq_mathkids') . '/js/flowplayer-3.2.8.min.js');
```
Comment (sdreher) [inline sites/default/themes/mythemes/acq_mathkids/template.php]: Can this commented code be removed also?

**Comment by nbuonin** [comment]
Got it, I removed those extra comments.

**Comment by nbuonin** [comment]
I cherry-picked the relevant commits to a new branch, which rewrites the history of this branch.  I'll close this so it doesn't get merged, which would be a Very Bad Thing.

**Pull request closed without merging**